### PR TITLE
Add a way to review only a certain reviewer todos

### PR DESCRIPTION
**Pull request by dlsniper**
Added a ` -i reviewerID ` as a flag for TODO so that I can filter only by the talks that I care about (the ones that were updated since my review). You can get the ID from the JSON file.

### CODE REVIEW

File: pc.go [hunk 1]
```diff
@@ -21,9 +21,10 @@ func main() {
 	sort := showCmd.Flag("sort", "sort by which column").Short('s').Default("rating").String()
 	reverse := showCmd.Flag("reverse", "reverse sort order").Short('r').Bool()
 	tag := showCmd.Flag("tag", "filter only tag").Short('t').String()
-	reviewersCmd := kingpin.Command("reviewers", "show reviewer compleition.")
+	reviewersCmd := kingpin.Command("reviewers", "show reviewer completion.")
 	revsort := reviewersCmd.Flag("sort", "sort by which column").Short('s').Default("count").String()
 	todoCmd := kingpin.Command("todo", "show outstanding proposals.")
+	reviewerID := kingpin.Flag("reviewer", "filter for reviwer ID").Short('i').Default("0").Int()
```
Comment (davecheney) [inline pc.go]: this should be todoCmd.Flag(...) so the flag is scoped to `pc todo`

File: show.go [hunk 2]
```diff
@@ -41,7 +41,7 @@ func show(id int, format string, ssort string, reverse bool, tag string) {
 	}
 
 	table := tablewriter.NewWriter(os.Stdout)
-	table.SetHeader([]string{"title", "format", "tags", "rating", "trust", "url"})
+	table.SetHeader([]string{"id", "title", "format", "tags", "rating", "trust", "submitted", "updated", "url"})
```
Comment (davecheney) [inline show.go]: this is going to wrap on all but the biggest screens. I don't care about the id field it's meaningless if you have the url

File: todo.go [hunk 3]
```diff
@@ -20,7 +20,7 @@ func todo(id int, all bool) {
 	check(err)
 
 	table := tablewriter.NewWriter(os.Stdout)
-	table.SetHeader([]string{"title", "format", "tags", "reason", "url"})
+	table.SetHeader([]string{"title", "format", "tags", "reason", "updated", "url"})
```
Comment (davecheney) [inline todo.go:23]: sgtm, but drop the reason field, we have no proposals that don't have reviews now.

**Review by davecheney — CHANGES_REQUESTED** [review]

**Comment by dlsniper** [comment]
Please let me do some cleanup. I can actually drop the first commit since we would have the ` todo -i reviewerID ` now to fill the gap I worked on from the first commit.

File: pc.go (hunk 1 above)
Comment (dlsniper) [reply pc.go]: Still learning kingpin, thanks.

File: todo.go (hunk 3 above)
Comment (dlsniper) [reply todo.go:23]: I'd leave the field there, for future users of this?

**Comment by dlsniper** [comment]
Updated to just have the todo command changed. I think/hope it's better now for users?

File: todo.go (hunk 3 above)
Comment (davecheney) [reply todo.go:23]: Yeah, that's fine. We won't need this tool in 24 hours.

**Review by davecheney — APPROVED** [review]

**Pull request merged**